pkg/controller/manager/manager_controller.go

```diff
@@ -305,6 +305,10 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		}
 	}
 
+	if certOperatorManaged && installation.CertificateManagement != nil {
```

Comment:
Is tlsSecret an user provided certificate ? From what I remember it should not be. Isn't it always generated by operator. This secret should be used within the k8s cluster for communication.

Reply:
It is, it is the only secret that we strongly encourage to byo independent of the features you use: https://docs.tigera.io/security/comms/manager-tls